rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -664,3 +711,274 @@ def test_applying_all_upgrade(globaldb: GlobalDBHandler):
     assert globaldb.get_setting_value('version', 0) == GLOBAL_DB_VERSION
     with globaldb.conn.cursor() as cursor:
         assert cursor.execute('SELECT COUNT(*) from assets WHERE identifier="eip155:/erc20:0x32c6fcC9bC912C4A30cd53D2E606461e44B77AF2"').fetchone()[0] == 1  # noqa: E501
+
+
+def test_update_consistency_with_packaged_db(
+        tmpdir_factory: 'pytest.TempdirFactory',
+        messages_aggregator: 'MessagesAggregator',
+):
+    """Test that the globalDB updates are consistent with the packaged one.
+    - All assets are present in both the cases.
+    - All details of these assets are the same in both the cases.
+    - All underlying assets are present and mapped in both the cases.
+    Protocol tokens that are queried automatically are not tested here."""
+    temp_data_dir = Path(tmpdir_factory.mktemp(GLOBALDIR_NAME))
+    (old_db_dir := temp_data_dir / GLOBALDIR_NAME).mkdir(parents=True, exist_ok=True)
+    request.urlretrieve(
+        url='https://github.com/rotki/rotki/raw/v1.16.0/rotkehlchen/data/global.db',
+        filename=old_db_dir / 'global.db',
+    )
+
+    globaldb = create_globaldb(data_directory=temp_data_dir, sql_vm_instructions_cb=0)
+
+    with (
+        globaldb.conn.read_ctx() as old_db_cursor,
+        globaldb.packaged_db_conn().read_ctx() as packaged_db_cursor,
+    ):
+        assert old_db_cursor.execute('SELECT value FROM settings WHERE name="assets_version"').fetchone()[0] == '1'  # noqa: E501
+        assert packaged_db_cursor.execute('SELECT value FROM settings WHERE name="assets_version"').fetchone()[0] == '24'  # noqa: E501
+
+    assets_updater = AssetsUpdater(msg_aggregator=messages_aggregator)
+    conflicts = assets_updater.perform_update(
+        up_to_version=None,
+        conflicts=None,
+    )
+    if conflicts is not None:
+        assert assets_updater.perform_update(
+            up_to_version=None,
+            conflicts={
+                Asset(conflict['identifier']): 'remote'
+                for conflict in conflicts
+            },
+        ) is None
+
+    with (
+        globaldb.conn.read_ctx() as old_db_cursor,
+        globaldb.packaged_db_conn().read_ctx() as packaged_db_cursor,
+    ):
+        ignored_identifiers = {
+            identifiers[0]
+            for cursor in (old_db_cursor, packaged_db_cursor)
+            for identifiers in cursor.execute(
+                'SELECT identifier FROM evm_tokens WHERE '
+                f'protocol IN ({",".join(["?"] * len(IGNORED_PROTOCOLS))}) OR '
+                'address in (SELECT value FROM general_cache)',
```

Comment:
if you use f string in one place we put it in all the string for consistency
```suggestion
                f'SELECT identifier FROM evm_tokens WHERE '
                f'protocol IN ({",".join(["?"] * len(IGNORED_PROTOCOLS))}) OR '
                f'address in (SELECT value FROM general_cache)',
```